src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -135,9 +239,9 @@
             background-color: lighten($ibexa-color-base-black, 7.5%);
```

Comment:
I don't know if it's a good idea to use some percent lighter colors

Reply:
It's old code, I don't want to style side-menu in this PR. I think it can be (and will be) changed later during redesign.